runtime/src/bank/partitioned_epoch_rewards/mod.rs

```diff
@@ -219,7 +215,6 @@ impl Bank {
     pub(super) fn force_partition_rewards_in_first_block_of_epoch(&self) -> bool {
         self.partitioned_epoch_rewards_config()
             .test_enable_partitioned_rewards
-            && self.get_reward_calculation_num_blocks() == 0
```

Comment:
I see, so this value was maybe made to be configurable just to support forcing rewards in the first block

Reply:
Yes, at least partly. I don't think that's sufficient reason to preserve the field, though.